src/preferences/dialog/dlgprefeffects.cpp

```diff
@@ -62,7 +62,8 @@ void DlgPrefEffects::slotApply() {
     for (EffectProfilePtr profile : m_availableEffectsModel.profiles()) {
         EffectManifestPointer pManifest = profile->pManifest;
         m_pEffectsManager->setEffectVisibility(pManifest, profile->bIsVisible);
-        m_pConfig->set(ConfigKey("[Visible Effects]", pManifest->id()), ConfigValue(profile->bIsVisible));
+        m_pConfig->set(ConfigKey("[Visible " + pManifest->backendName() + " Effects]", pManifest->id()), 
```

Comment:
Add a comment here explaining why we put the backendName in the ConfigKey group.